python/nano/src/setup.py

```diff
@@ -154,5 +153,7 @@ def setup_package(plat_name):
                          f"--plat-name should be one of {valid_plat_names}"
                          f" but got {verbose_plat_name}")
     plat_name = verbose_plat_name
+    sys.argv.remove(sys.argv[idx+1])
```

Comment:
what these lines are for

Reply:
I think I didn't make any changes of this line? Anyway, I will fix it.

Reply:
Done, my fault, I will review the modifications when I push commits in the future.